lib/etl/transform/transform_json.rb

```diff
@@ -0,0 +1,38 @@
+require 'json'
+require 'json-schema'
+
+class TransformJson
+  def process(input_json)
+    @input_json = input_json
+    pl = parsed_list
+    JSON::Validator.validate(schema_fname, pl)
+    pl
+    
+  rescue => error
+    ap "Error opening json file: #{error}"
+  end
+
+  private
+
+  attr_reader :input_json, :schema_fname
+  
+  def schema_fname
+    raise NotImplementedError.new("#{self.class.name}#area is an abstract method.")
+  end
+  
+  def parsed_list
+    JSON.parse(File.read(input_json), :symbolize_names => true)
```

Comment:
You can change this method to memoize the return value (so it doesn't do a file read twice) like so:

```
def parsed_list
  @_parsed_list ||= JSON.parse(File.read(input_json), symbolize_names: true)
end
```

That way, in other locations, you can just access `parsed_list` instead of assigning the value to return variable.

Also, in my comment here I've used the more concise way of defining a symbol, changing `:symbolize_names => true` to `symbolize_names: true`.